common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -5508,6 +5508,17 @@ public static enum ConfVars {
     LLAP_TASK_TIME_SUMMARY(
         "hive.llap.task.time.print.summary", false,
         "Display queue and runtime of tasks by host for every query executed by the shell."),
+
+    LLAP_LOCAL_DIR_CLEANER_CLEANUP_INTERVAL(
```

Comment:
Can we reuse the existing cleaning feature?
https://github.com/apache/hive/blob/e892ce7212b2f0c9e55092d89076b129c86c8172/common/src/java/org/apache/hadoop/hive/conf/HiveConf.java#L3894-L3898

Reply:
I'll check: maybe the utility itself can be reused but with keeping a separate config

Reply:
checked [ClearDanglingScratchDir](https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/session/ClearDanglingScratchDir.java), it seems to be specific to scratch dir cleanup and it's too smart for this scenario, I feel it would be more complicated to make it general and reuse than having a simple utility for the cases: ClearDanglingScratchDir checks lock files, works on hdfs, has a command line utility, etc., whereas LocalDirCleaner is just about to remove old files from a local folder

Reply:
Ok,will come back to check this after vacation. :)